backend/src/services/qrCode.ts

```diff
@@ -1,9 +1,96 @@
+import { QrCode } from '../models/qrCode';
 import { db } from './firebase'
+import nodeConfig from 'config';
+import Logger from '../utils/logger';
+import Joi from 'joi';
 
 export const getQrCode = async (qid: string) => {
-    let doc = await db.collection('qr-codes').doc(qid.toString()).get()
+    let doc = await db.collection(nodeConfig.get('collections.qrcodes')).doc(qid.toString()).get()
```

Comment:
if you are fetching collection names from the config file then make sure the values exits in them.

Reply:
add an entry in the collections object in both the prod and dev config files

Reply:
```
"collections":{
    "qrcodes": "qr-codes"
}
```